package.js

```diff
@@ -3,6 +3,7 @@ Package.describe({
   summary: 'Materialized connection status.',
   version: '0.0.2',
   git: 'https://github.com/modweb/meteor-materialize-connection-status'
+  version: '0.0.3',
```

Comment:
Oops, sorry. I had other changes in here that I didn't want to include in the pull request (to publish my own fork -- I didn't expect you to merge it so quickly!). I should know better than to manually edit diffs.